onnxruntime/test/autoep/library/example_plugin_ep_utils.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Note: Moved random utility functions from `example_plugin_ep.cc` into this new utils file as the example EP code was getting a bit long.